website/server/docgenHelpers.js

```diff
@@ -1,5 +1,12 @@
 "use strict";
-var b = require('react-docgen/node_modules/recast').types.builders;
+var b;
+try {
+  // npm 3 probably
+  b = require('recast').types.builders;
```

Comment:
Oh I see. Should package.json in `website` depend on recast then? (Or is this a weird case where recast is stateful and different packages need to access the same instance?)